tests/ui/abi/invalid-call-abi.rs

```diff
@@ -0,0 +1,14 @@
+// Fix for #142969.
+
+#![feature(abi_c_cmse_nonsecure_call)]
+
+const extern "C-cmse-nonsecure-call" fn foo() {
```

Comment:
I am unfortunately unaware of the internals of this ABI, but do I need to limit this test's targets so that it doesn't pass where it shouldn't?

Reply:
Also, I said this in zulip but I'd really like `extern "rustc-invalid"` which is an always-invalid ABI for testing purposes, for things just like this.